src/components/file-uploader/__snapshots__/UIFileUploader.test.ts.snap

```diff
@@ -37,7 +37,7 @@ exports[`UIFileUploader should display correctly 1`] = `
          or 
       </p>
       <button
-        class="border-brand-400 text-white bg-brand-400 ring-brand-700 hover:bg-brand-500 hover:border-brand-500 border px-3.5 py-2 focus:ring-2 ui-button rounded font-bold text-sm m-0 focus:outline-none ring-0 disabled:cursor-not-allowed mt-2"
+        class="border-brand-600 text-white bg-brand-600 ring-brand-500 hover:bg-brand-700 hover:border-brand-700 shadow-sm border px-3.5 py-2 focus:ring-2 ui-button rounded font-bold text-sm m-0 focus:outline-none ring-0 disabled:cursor-not-allowed disabled:shadow-none mt-2"
```

Comment:
No, but it seems that the guide was wrong, because apps are implementing their own buttons with a slightly more vibrant colour. So the mockups didn't use the style guide or the styleguide is wrong. This change and the table headers is why I want @sixteenstudio to load up the app and review the styles